docs/src/componentDocs/Header/markdown/HeaderExamples.mdx

```diff
@@ -1,180 +1,35 @@
-import Box from '@mui/material/Box';
-import headerImageExample from '../images/headerImage.png';
-import headerBackgroundExample from '../images/headerBackgroundImage.png';
-import headerActionsExample from '../images/headerActionsImage.png';
-import headerSearchExample from '../images/headerSearchImage.png';
-import headerCollapsedExample from '../images/headerCollapsedImage.png';
-import headerExpandedExample from '../images/headerExpandedImage.png';
+import {Header} from '../examples/Header';
+import {HeaderWithBackground} from '../examples/HeaderWithBackground';
+import {HeaderWithActions} from '../examples/HeaderWithActions';
+import {HeaderWithSearch} from '../examples/HeaderWithSearch';
+import {HeaderWithExpandCollapse} from '../examples/HeaderWithExpandCollapse';
 
 ## Header
 
-A basic `Header` component is used at the top of the screen and can show a title and (optional) subtitle. A third line of text can also be configured using the `info` prop, but it will only be visible when the `Header` is expanded (see below). 
+A basic `Header` component is used at the top of the screen and can show a title and (optional) subtitle. A third line of text can also be configured using the `info` prop, but it will only be visible when the `Header` is expanded (see below).
 
-<Box
-  sx={{
-    width: '100%',
-    textAlign: 'center',
-  }}>
-  <Box
-    sx={{
-      width: '100%',
-      maxWidth: '500px',
-    }}
-    component="img"
-    alt="Header"
-    src={headerImageExample}
-  />
-</Box>
-
-```
-<Header
-    title={'Valley Forge'}
-    subtitle={'The Last Stand'}
-/>;
-```
+<Header />
 
 ## Changing the Background
 
 You can apply a simple background to the Header using the `backgroundColor` prop. You can also apply an image background using `backgroundImage`. If both are applied, they will be blended together.
 
-<Box
-  sx={{
-    width: '100%',
-    textAlign: 'center',
-  }}>
-  <Box
-    sx={{
-      width: '100%',
-      maxWidth: '500px',
-    }}
-    component="img"
-    alt="Header background image"
-    src={headerBackgroundExample}
-  />
-</Box>
-
-```
-<Header
-    title={'Valley Forge'}
-    subtitle={'The Last Stand'}
-    backgroundImage={farm.jpg}
-/>;
-```
+<HeaderWithBackground />
```

Comment:
The code snippet has commented prop `backgroundImage`

<img width="1792" alt="Screenshot 2024-06-17 at 1 07 21 PM" src="https://github.com/etn-ccis/blui-react-native/assets/120575281/2c887fec-fb3c-4297-90f4-0dba2db27ad9">